graphql/batch_test.go

```diff
@@ -778,9 +778,6 @@ func TestBatchFieldFuncExecution(t *testing.T) {
 
 				var e graphql.ExecutorRunner
 				e = graphql.NewBatchExecutor(graphql.NewImmediateGoroutineScheduler())
-				if cond == OldExecutor {
```

Comment:
Delete the "OldExecutor" Constant plz

Reply:
Actually, nvm, I'm going to land https://github.com/samsarahq/thunder/pull/264, you can just remove this change and rebase on top of it.